snmp_dell/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: snmp_Dell
+
+## Overview
+
+This check monitors [Dell][1].
```

Comment:
@naaltman would need your input here

Reply:
Change to: 

Dell Inc. is a technology company that develops computers and network hardware for high performance networking in both small large businesses. Collect metrics for monitoring and alerting from Dell hardware including: 

- Dell PowerEdge
- Dell iDRAC
- Dell EMC Isilon 

For a complete list of all the metrics collected from Dell devices, see the Network Device Monitoring documentation (https://docs.datadoghq.com/network_performance_monitoring/devices/data)